src/components/toast/components/close-button/close-button.module.css

```diff
@@ -0,0 +1,49 @@
+.closeButton {
+  composes: g-focus-ring-from-box-shadow from global;
+
+  --background-color: transparent;
+  --border-color: transparent;
+  --color: var(--token-color-foreground-faint);
+  --size: 24px;
+
+  appearance: none;
+  background: none;
+  border: none;
+  border-radius: 5px;
+  color: var(--color);
+  cursor: pointer;
+  isolation: isolate;
+  display: grid;
+  height: var(--size);
+  margin: 0;
+  padding: 0;
+  place-items: center;
+  position: relative;
+  width: var(--size);
+
+  &::before {
+    background-color: var(--background-color);
+    border-radius: 5px;
+    border: 1px solid var(--border-color);
+    content: '';
+    height: 100%;
+    left: 0;
+    position: absolute;
+    top: 0;
+    width: 100%;
+    z-index: -1;
+  }
+
+  &:hover {
+    /* Note: use of alpha-100 differs from HDS toast implementation
+       (https://github.com/hashicorp/design-system/blob/f1d2475d86acb769833504cfb13b3531b387ae7e/packages/components/app/styles/components/alert.scss#L149).
+       Have done this to avoid using a non-token value. May need to adjust. */
+    --background-color: var(--token-color-palette-alpha-100);
```

Comment:
See comment here, open to other approaches. Intent here is to try to get _something_ approximately right rendering; I poked in [the HDS Figma file for Alert & Toast](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?node-id=1377%3A11987) but didn't see a hover state (relatively quick look so I may be missing something). Given lack of clear spec this seems very much close enough IMO.

Reply:
This seems fine to me! Also reasonable to ask about this in #team-design-systems. 😊